### PR TITLE
fix: scrna gene exp violin disables brushing and label menu

### LINKED ISSUE #2438: violin plot using scrna gene exp data, `List sample` over a range can list all cells rather by range
right now on scrna gene exp violin, it has the original capability to select a range to `List samples`. if this function is to be kept, it should be called `List cells`. however, this function will result in all cells being loaded without observing exp value range

### PR DESCRIPTION
**Pull request by xzhou82**
## Description

closes #2438 
violin CI passes, scrna views (native and gdc) works

## Checklist

[Check each task](https://github.com/stjude/proteinpaint/wiki/Pull-Request-Checklist) that has been performed or verified to be not applicable.
- [x] Tests: added and/or passed unit and integration tests, or N/A
- [x] Todos: commented or documented, or N/A
- [x] Notable Changes: updated release.txt, prefixed a commit message with "fix:" or "feat:", added to an internal tracking document, or N/A

### CODE REVIEW

**Comment by karishma-gangwani** [comment]
thanks, can you add link or steps on how to test it

File: client/plots/violin.interactivity.js [hunk 1]
```diff
@@ -25,9 +25,8 @@ export function setInteractivity(self) {
 	}
 
 	self.displayLabelClickMenu = function (t1, t2, plot, event) {
-		if (!t2 || self.data.plots.length === 1) {
-			return
-		}
+		if (!t2 || self.data.plots.length === 1) return // when no term 2 and just one violin, do not show options on the sole violin label
```
Comment (airenzp) [inline client/plots/violin.interactivity.js:28]: you may want to see the t1 values, @karishma-gangwani requested this feature.

**Review by airenzp — APPROVED** [review]
I am approving, thanks for the fix. I would suggest to allow showing the sample values even when there is no overlay.

**Comment by airenzp** [comment]
@karishma-gangwani you can test this on any SC plot, select the gene expression tab, search a gene, then open the violin associated.

**Pull request merged**